scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -10,6 +10,11 @@
     ARDANA_INIT_AUTO: 1
 
   tasks:
+  - name: Write deployer info to /etc/motd
+    become: yes
+    shell: |
+      echo "DEPLOYER_IP={{ deployer_floating_ip | default(None) }}" > /etc/motd
```

Comment:
isnt there some idempotent ansible module? similar to https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.append

Reply:
I guess that lineinfile can be a good match here http://docs.ansible.com/ansible/latest/lineinfile_module.html

```
- name: Write deployer fip to /etc/motd
  become: yes
  lineinfile:
    dest: /etc/motd
    state: present
    line: 'DEPLOYER_IP={{ deployer_floating_ip | default(None) }}'
```